gammapy/analysis/config.py

```diff
@@ -130,16 +143,19 @@ class BackgroundConfig(GammapyBaseConfig):
     method: BackgroundMethodEnum = BackgroundMethodEnum.reflected
     exclusion: FilePath = None
 
+class SafeMaskSettingsConfig(GammapyBaseConfig):
+    offset_max: AngleType = "2.5 deg"
+    aeff_percent: float = 10
+    bias_percent: float = 10
+
+class SafeMaskConfig(GammapyBaseConfig):
+    methods: List[SafeMaskMethodsEnum] = [SafeMaskMethodsEnum.aeff_default]
+    settings: SafeMaskSettingsConfig = SafeMaskSettingsConfig()
 
 class EnergyAxesConfig(GammapyBaseConfig):
     energy: EnergyAxisConfig = EnergyAxisConfig()
     energy_true: EnergyAxisConfig = EnergyAxisConfig()
 
-
-class SelectionConfig(GammapyBaseConfig):
-    offset_max: AngleType = "2.5 deg"
-
-
```

Comment:
The `offset_max` is also used elsewhere in `core.py` to produce cutouts of datasets.  Its value is fetched from `datasets.geom.selection` settings. 

Does it make sense to have two different values for the offset? One used to produce cutouts of datasets in the analysis process and another one related with the safe mask?  If we keep two different params for the offset, they could have the same names since they would at different levels in the settings.

For example:
-  `datasets.geom.selection.offse_max`
-  `datasets.safe_mask.settings.offset_max`

Reply:
Normally I removed the `datasets.geom.selection.offset_max`to keep only one. And now the `MapDataset.cutout` use the value in `datasets.safe_mask.setting.offset_max` in `Analysis`.
The code should be changed in such a way that if `SafeMaskMaker` does not use the `offset-max` method, it shouldn't do a cutout.

 I understand the logic of the `cutout` to be directly connected to the `offset_max` mask. 
If we have no `offset-max`, we probably shouldn't be doing any cutout no?

Reply:
In general I think the  `offset_max` and cutout width should be two different parameters. The first argument is handling of PSF leakage, so the cutout should be larger then the `offset_max` to account for PSF leakage in joint analyses. Secondly the offset_max can be changed after data reduction (before stacking) when the cutout is larger.

However the cutout-width doesn't have to be configurable by the user from the HLI. So I'm fine to have only one `offset-max` parameter in the HLI and derive internally the cutout-width in a reasonable way...

Reply:
The `SelectionConfig` is back in.